### PR TITLE
feat(LLM/send): New gas fees parent account error

### PR DESCRIPTION
**Pull request by mle-gall**
### 📝 Description

New insufficient funds errors when selecting a token for which the parent account doesn't have enough balance to pay gas fees + partial visual rework of the gas fees selector (colors & border radii)

https://github.com/LedgerHQ/ledger-live/assets/39890664/b50b77a3-8f5f-473a-8b4c-d6b1437a0d83

### ❓ Context

- **JIRA or GitHub link**:  [LIVE-9792]

### ✅ Checklist

Pull Requests must pass the CI and be code reviewed. Set as Draft if the PR is not ready.

- [ ] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - 

---

### 🧐 Checklist for the PR Reviewers

- [ ] **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- [ ] **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- [ ] **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- [ ] **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- [ ] **Any new dependencies** have been justified and documented.
- [ ] **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

[LIVE-9792]: https://ledgerhq.atlassian.net/browse/LIVE-9792?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx [hunk 1]
```diff
@@ -75,6 +86,10 @@ export default function SelectFeesStrategy({
     () => setNetworkFeeHelpOpened(!isNetworkFeeHelpOpened),
     [isNetworkFeeHelpOpened],
   );
+  const navigation = useNavigation();
+
+  const errors = status?.errors;
+  const insufficuentError = errors?.[Object.keys(errors)[0]] || null;
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx:92]: Isn't it the same as `Object.values(errors)[0]` ?

File: apps/ledger-live-mobile/src/families/evm/SelectFeesStrategy.tsx [hunk 2]
```diff
@@ -81,6 +87,10 @@ export default function SelectFeesStrategy({
     () => setNetworkFeeHelpOpened(!isNetworkFeeHelpOpened),
     [isNetworkFeeHelpOpened],
   );
+  const navigation = useNavigation();
+
+  const errors = status?.errors;
+  const insufficuentError = errors?.[Object.keys(errors)[0]] || null;
```
Comment (cgrellard-ledger) [inline apps/ledger-live-mobile/src/families/evm/SelectFeesStrategy.tsx:93]: Same

File: apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx [hunk 3]
```diff
@@ -169,19 +202,26 @@ export default function SelectFeesStrategy({
...
           }
         >
           {null}
         </SummaryRow>
-
+        {insufficuentError && (
+          <TouchableOpacity onPress={() => onBuy(mainAccount)}>
+            <Alert
+              type="warning"
+              title={(<TranslatedError error={insufficuentError} />) as unknown as string}
```
[the remark this answers is not in the document]
Comment (mle-gall) [reply apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx:221]: No idea why. I had a TS error there but not on the other file...
TS is sometimes driving me crazy.

[the remark this answers is not in the document]
Comment (KVNLS) [reply apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx:221]: It comes from the type defined for `title: string;`. If we want to accept a React component as title props we need to update the type in Alert component.

File: apps/ledger-live-mobile/src/components/SendRowsFee.tsx [hunk 4]
```diff
@@ -34,22 +34,24 @@ export default ({
   navigation,
   route,
   setTransaction,
+  status,
   ...props
 }: Props) => {
   const mainAccount = getMainAccount(account, parentAccount);
 
   // eslint-disable-next-line no-prototype-builtins
   if (perFamily.hasOwnProperty(mainAccount.currency.family)) {
```
Comment (KVNLS) [inline apps/ledger-live-mobile/src/components/SendRowsFee.tsx:43]: I know it was already like this but could you fix this eslint error, please?
```suggestion
  const hasFamilyProperty = Object.prototype.hasOwnProperty.call(
    perFamily,
    mainAccount.currency.family,
  );

  if (hasFamilyProperty) {
```

File: libs/coin-evm/src/getTransactionStatus.ts [hunk 5]
```diff
@@ -158,7 +159,16 @@ const validateGas = (
   ) {
     errors.gasPrice = new FeeNotLoaded(); // "Could not load fee rates. Please set manual fees"
   } else if (tx.recipient && estimatedFees.gt(account.balance)) {
-    errors.gasPrice = new NotEnoughGas(); // "The parent account balance is insufficient for network fees"
+    const query = new URLSearchParams({
+      ...(account?.id ? { account: account.id } : {}),
+    });
+    errors.gasPrice = new NotEnoughGas(undefined, {
+      // "You need {{fees}} {{ticker}} for network fees to swap as you are on {{cryptoName}} network. <link0>Buy {{ticker}}</link0>"
```
Comment (KVNLS) [inline libs/coin-evm/src/getTransactionStatus.ts:166]: 🗑️

**Review by KVNLS — CHANGES_REQUESTED** [review]
How do we know that only Bitcoin and evm are impacted?

**Review by KVNLS — APPROVED** [review]

**Pull request merged**